CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Deprecate the `logger` option (#1167)
 - Pass the event hint from the `capture*()` methods down to the `before_send` callback (#1138)
+- Deprecate the `tags` option (#1174)
```

Comment:
```suggestion
- Deprecate the `tags` option, see the [docs](https://docs.sentry.io/platforms/php/guides/laravel/enriching-events/tags/) for other ways to set tags (#1174)
```

Not sure this is the correct way but I like adding a little bit of “upgrade docs” to the changelog. Could also be mentioned in the 4.0 upgrade docs instead of here so people can prepare and use the new methods already.

Reply:
>Could also be mentioned in the 4.0 upgrade docs

I don't know if it's would be correct for a few reaons:

- People that already upgraded will likely missing the new changes to that document
- The old way of setting the tags using the options works and it's fine until `5.0`, so at best this should be reported in a `UPGRADE-5.0.md` document

Reply:
4 is the next release right? Not 5? Otherwise your deprecation exceptions are also wrong 😅

It should be mentioned in the upgrade docs for whatever version the tags option is actually removed, let's say it like that 😬

Reply:
>4 is the next release right? Not 5?

Yeah sorry, my mistake typing the message